.github/workflows/e2e-tests.yml

```diff
@@ -17,11 +16,21 @@ jobs:
           regex_flags: 'gm'
           search_string: ${{github.event.deployment.environment}}
       - name: Get deployment url
-        id: deployment_url
-        env:
-          DEPLOYMENT_URL: 'https://lightdash-pr-${{steps.regex.outputs.first_match}}.onrender.com'
-        run:
-          echo "::set-output name=url::$DEPLOYMENT_URL"
+        id: deployment-url
+        run: echo "url=https://lightdash-pr-${{steps.regex.outputs.first_match}}.onrender.com" >> "$GITHUB_OUTPUT"
+
+  app-tests:
+    runs-on: ubuntu-latest
+    needs: prepare-preview
+    container:
+      image: cypress/browsers:node18.12.0-chrome107
+      options: --user 1001
```

Comment:
I have no idea what any of this means :upside_down_face:

Reply:
recommended container for cypress and option is needed to avoid some permission errors. We already use this in the current github actions.